### PR TITLE
Final PR - Ruby on Rails Group project - Recipe app

### PR DESCRIPTION
**Pull request by jlberbesi**
Hello fellow code reviewer, In this project, we have created a recipe app that allows to record recipes, create a list of foods, and generate a shopping list to buy the foods the user needs in order to cook the recipes the user wants.

We did the following tasks:

Created routes for all the pages.
Created the data base tables for all the classes needed.
Created the models for the users, foods, recipes and recipe foods.
Created the controllers for the users, foods, recipes and recipe foods.
Install Devise gem to allow authentication.
Created the views for registration, confirmation, log in and edit account.
Created the view for the list and to create foods.
Created the view for the private and public recipes.
Created the views for the shopping list.
Added styles to all the views.
Added unit and integrations tests using Rspec and Capybara.

### CODE REVIEW

File: app/views/public_recipes/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            <h3><%= recipe.name %></h3>
+            <p><strong>By:</strong> <%= recipe.user.name %></p>
+          </div>
+          <div class="right-content">
+            <p><strong>Total Food Items:</strong> <%= recipe.foods.count %></p>
+            <p><strong>Total Price:</strong> <%= calculate_total_price_from_recipe_foods(recipe.recipe_foods) %></p>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
```
Comment (Olamarx) [inline app/views/public_recipes/index.html.erb:23]: - [ ] Good job so far, however, there should be a click functionality in the public recipe page as required, at the moment, I can only see a hover effect. Kindly add it.
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- All test cases passed ✔️
- Readme file professional ✔️
- Good use of Git and GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/public_recipes/index.html.erb (hunk 1 above)
Comment (jlberbesi) [reply app/views/public_recipes/index.html.erb]: imagen

imagen

Done!!!

**Review by OyePriscilla — APPROVED** [review]
Hi @jlberbesi and @CobbyElsonfx 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
 


## Highlights
Clicking on a recipe item leads to recipe details. ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**